src/main/kotlin/minesweeper/entity/MineCounting.kt

```diff
@@ -0,0 +1,28 @@
+package minesweeper.entity
+
+class MineCounting(private val minedMap: List<List<MapElement>>, private val mapInfo: MapInformation) {
+    val dx = listOf<Int>(0, 1, 0, -1)
```

Comment:
* Int 타입은 생략이 가능해 보여요 :) 
* private으로 선언이 가능해보여요!

```suggestion
    private val dx = listOf(0, 1, 0, -1)
```